api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -76,10 +78,20 @@ func (mw *azureManagedControlPlaneWebhook) Default(ctx context.Context, obj runt
 		networkPlugin := "azure"
 		m.Spec.NetworkPlugin = &networkPlugin
 	}
+
 	if m.Spec.LoadBalancerSKU == nil {
-		loadBalancerSKU := "Standard"
+		loadBalancerSKU := "standard"
 		m.Spec.LoadBalancerSKU = &loadBalancerSKU
 	}
+	lbSku := ptr.Deref(m.Spec.LoadBalancerSKU, "")
+	if lbSku == "Standard" {
```

Comment:
Does this need to be uncapitalized? I noticed standard and basic is lowercase in other places, such as when `loadBalancerSKU` is set on line 83.

Reply:
That's what this is accounting for, if I'm understanding your question right. Before the CAPZ API only accepted `Standard` but the ASO API only accepts `standard`. This change translates the incorrect `Standard` to `standard`.